socorro/unittest/external/es/test_crashstorage.py

```diff
@@ -368,10 +335,13 @@ def test_success(self, espy_mock):
         # IndexCreator but is part of the crashstorage workflow).
         assert espy_mock.client.IndicesClient.called
 
+        expected_processed_crash = deepcopy(a_processed_crash)
+        reconstitute_datetimes(expected_processed_crash)
```

Comment:
Previously, this test was order dependent. Something was changing the processed crash and then that change carried on through other tests. After fixing that by deepcopying in every case, suddenly the expected document didn't match the actual document because the actual document has datetime strings turned into datetimes. So ew have to do this thing here now to get everything to match.